system/HTTP/DownloadResponse.php

```diff
@@ -269,7 +269,7 @@ public function getReason(): string
 	 */
 	public function setDate(\DateTime $date)
 	{
-		$date->setTimezone(new \DateTimeZone('UTC'));
+		$date->setTimezone(new \DateTimeZone(app_timezone() ?? 'UTC'));
 
 		$this->setHeader('Date', $date->format('D, d M Y H:i:s') . ' GMT');
```

Comment:
If the `app_timezone()` is `America/Sao_Paulo` it is not `GMT`. [See also](https://tools.ietf.org/html/rfc7231#section-7.1.1.1).